src/msix/msix.cpp

```diff
@@ -17,7 +18,10 @@
 #include "AppxPackageObject.hpp"
 #include "MsixFeatureSelector.hpp"
 #include "AppxPackageWriter.hpp"
+#include "AppxBundleWriter.hpp"
 #include "ScopeExit.hpp"
+#include "MSIXWindows.hpp"
+#include "GeneralUtil.hpp"

```

Comment:
remove

Reply:
I removed the MSIXWindows.hpp file but I need the GeneralUtil.hpp file to be there because it has helper functions to process the input in the PackBundle method.